clients/commons/services/api/apiMethods.ts

```diff
@@ -0,0 +1,30 @@
+import api from '../../config/apiConfig'
+import { AxiosResponse } from 'axios'
+
+export const get = async(url: string, params?: object): Promise<AxiosResponse> => {
+  return await api.get(url, { params })
```

Comment:
bro i thought we removed the extra await

Reply:
if i remove it, it gives linting issues

Reply:
Updated in 822b3b468a429c8071ce249b63e8ab0ca5ed0cc5